Anti-Raid / Panic mode

zaidi-01 opened the pull request:
Anti-Raid prevents spam / raid while it is enabled. The "panic" command updates a global variable(raid_mode) which tells the "on_message" event whether it is on or off.

zaidi-01 committed:
Added Anti-Raid mode in 'on_message' event

Anti-Raid mode allows an admin to prevent spam that could happen in the server

zaidi-01 committed:
Delete slnx.sqlite

zaidi-01 committed:
Delete ProjectSettings.json

zaidi-01 committed:
Delete .suo

BlackTiger1500 commented:
Anti-Raid prevents spam / raid while it is enabled. The "panic" command updates a global variable(raid_mode) which tells the "on_message" event whether it is on or off.

zaidi-01 committed:
Updated 'raid_mode' to have a dictionary

Anti-Raid will use a dictionary containing all the channels that are being raided

tom-sherman commented:
This code doesn't even run, I'll close and you can redo when the new db and config changes are in
